CMakeLists.txt

```
@@ -241,6 +246,10 @@ if("${DDC_BUILD_KERNELS_SPLINES}")
   find_package(Ginkgo 1.7.0 EXACT REQUIRED)
   target_link_libraries(DDC INTERFACE Ginkgo::ginkgo)
   target_compile_definitions(DDC INTERFACE ginkgo_AVAIL)
+
+  # Lapack
+  find_package(LAPACK REQUIRED COMPONENTS CXX)
```

```suggestion
  find_package(LAPACK REQUIRED)
```

We also need  to fix this in Gysela, there is no such component in `FindLAPACK.cmake`